src/V3Tristate.cpp

```diff
@@ -960,6 +960,21 @@ class TristateVisitor final : public TristateBaseVisitor {
                 if (debug() >= 9) newp->dumpTree(cout, "-caseeq-new: ");
                 nodep->replaceWith(newp);
                 VL_DO_DANGLING(pushDeletep(nodep), nodep);
+            } else if (constp && nodep->rhsp()->user1p()) {
+                FileLine* const fl = nodep->fileline();
+                constp->unlinkFrBack();
+                AstNode* rhsp = nodep->rhsp()->unlinkFrBack();
+                AstNode* newp = new AstLogAnd(
```

Comment:
```suggestion
                AstNode* newp = new AstLogAnd{
```

Please use {} style constructors on all new/modified code.

Reply:
Done.